scripts/buildThemeTypings.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Rename .js files to .ts
+for file in packages/**/theme/**/*.js
+do
+  mv "$file" "${file%.js}.ts"
+done
+
+# Build type definitions
+npm run build:ts
+
+# Restore original .js files
+git checkout packages/**/theme/**/*.js
+
+# Remove .ts files but keep .d.ts
+npx del-cli "packages/**/theme/**/*.ts" "!packages/**/theme/**/*.d.ts"
```

Comment:
Looks like `rimraf` doesn't support excluding files using glob, so I had to use `del-cli` instead.

Reply:
This seems to do the same:
```
find . -path "./packages/*/theme/*.ts" ! -name "*.d.ts" -type f -delete
```